spring-cloud-gcp-data-spanner/src/test/java/org/springframework/cloud/gcp/data/spanner/core/convert/MappingSpannerConverterTests.java

```diff
@@ -99,13 +99,10 @@ public void canConvertDefaultTypesCustomConverters() {
 	private void verifyCanConvert(MappingSpannerConverter converter, Class javaType,
 			Class spannerType) {
 		MappingSpannerWriteConverter writeConverter = converter.getWriteConverter();
-		MappingSpannerReadConverter readConverter = converter.getReadConverter();
+		ConverterAwareMappingSpannerEntityReader readConverter = converter.getEntityReader();
 
-		assertTrue(converter.canConvert(javaType, spannerType));
-		assertTrue(converter.canConvert(spannerType, javaType));
-
-		assertTrue(writeConverter.canConvert(javaType, spannerType));
```

Comment:
Why were these two lines removed?
`assertTrue(writeConverter.canConvert(javaType, spannerType));`
`assertTrue(readConverter.canConvert(spannerType, javaType));`

These are supposed to confirm that the read and write sub converters can handle the necessary conversions in the read and write directions, respectively.